### ISSUE TITLE
add efficientnet_b0

### ISSUE THREAD

**Issue by sukyoungjeong-furiosa** [furiosa-ai/furiosa-models#107]
This is a tracking issue for adding model: `efficientnet_b0`

## Checklist
- [x]  Identify the model structure.
- [x]  Perform quantization and calibration of the model.
  - [x] https://github.com/furiosa-ai/furiosa-models/pull/121
  - Accuracy: `16.104%` (original f32 model: `77.698%`)
- [x]  Deploy an initial model implementation, along with compiled IRs and sanity-checking tests.
- [x]  Implement pre/post-processing codes and regression tests.
- [x]  Add unit tests, example code, preliminary documentation.
- [ ]  Refine and improve the overall code structure.
- [x]  Enhance and proof-read documentation.

## About the model
- model source: [`rwightman/gen-efficientnet-pytorch`](https://github.com/rwightman/gen-efficientnet-pytorch)
- ```python
  import torch
  import onnx
  from tempfile import NamedTemporaryFile
  import furiosa.optimizer.frontend.onnx

  torch_model = torch.hub.load(
      "rwightman/gen-efficientnet-pytorch", "efficientnet_b0", pretrained=True, trust_repo=True
  )
  torch_model = torch_model.eval()
  with NamedTemporaryFile(suffix=".onnx") as file:
      torch.onnx.export(
          torch_model, torch.randn(1, 3, 224, 224), file, opset_version=13
      )
      onnx_model = onnx.load_model(file.name)

  onnx.checker.check_model(onnx_model, full_check=True)
  optimized_onnx_model = furiosa.optimizer.frontend.onnx.optimize_model(onnx_model)
  onnx.checker.check_model(optimized_onnx_model, full_check=True)
  optimized_onnx_model = optimized_onnx_model.SerializeToString()
  ```
- input: 3-channel (BGR) image of 224x224 `numpy.float32` (Memory format: NCHW)
- output: `numpy.float32` tensor with the shape ([1,1000])
- accuracy: `16.104%` (original f32 model: `77.698%`)
